app/models/credential.rb

```diff
@@ -17,6 +17,10 @@ def login
     jwt_payload.fetch("login")
   end
 
+  def current_user_id
```

Comment:
Perhaps just `user_id`? The "current" makes sense in a controller clarify which user, but a credential always is about zero or one users.

Reply:
i'm moving this method to the panoptes client in a PR i'm working on now; it'll be called authenticated_user_id there